assets/src/domain/eventEditor/ui/datetimes/datesList/actionsMenu/AssignTicketsButton.tsx

```diff
@@ -21,12 +21,11 @@ const AssignTicketsButton: React.FC<EntityListItemProps<Datetime>> = React.memo(
 		entity: 'datetimes',
 		entityId: entity.id,
 	});
-	const count = relatedTickets.length;
 
-	const relatedTicketDbIds = relatedTickets.map(({ dbId }) => dbId);
+	const count = relatedTickets.length;
 
 	const title = count
-		? `${__('Related Tickets:')} ${relatedTicketDbIds.join(', ')}`
+		? `${__('Number of related tickets:')} ${count}`
```

Comment:
we don't need the count here because that is already shown in the item count bubble, we just need the text to explain what the number in the bubble represents.

So this just needs to be:

```suggestion
		? __('Number of related tickets')
```